File: app/src/main/java/jp/tetra2000/androidprocessingtest/MainActivity.java
package jp.tetra2000.androidprocessingtest;

import android.app.Activity;
import android.app.Fragment;
import android.app.FragmentManager;
import android.os.Bundle;

public class MainActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        FragmentManager fragmentManager = getFragmentManager();
        Fragment fragment = new Sketch();
        fragmentManager.beginTransaction()
                .replace(R.id.container, fragment)
                .commit();
    }
}

File: app/src/main/java/jp/tetra2000/androidprocessingtest/MainActivity.kt
package jp.tetra2000.androidprocessingtest

import android.app.Activity
import android.os.Bundle

class MainActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val fragmentManager = fragmentManager
        val fragment = Sketch()
        fragmentManager.beginTransaction().replace(R.id.container, fragment).commit()
    }

    private fun setupCamera() {
    }
}
